zcash_client_sqlite/src/lib.rs

```diff
@@ -59,10 +60,17 @@ use zcash_client_backend::{
 
 use crate::error::SqliteClientError;
 
+use {
+    zcash_client_backend::wallet::WalletTransparentOutput,
+    zcash_primitives::legacy::TransparentAddress,
+};
+
 pub mod chain;
 pub mod error;
 pub mod wallet;
 
+pub const PRUNING_HEIGHT: u32 = 100;
```

Comment:
Either make this constant private, or document it. Personally I think both, as AFAICT this constant is only used internally.